inc/lazyload_replacer.php

```diff
@@ -200,6 +200,8 @@ public function can_lazyload_for( $url, $tag = '' ) {
 		if ( false === Optml_Filters::should_do_image( $url, self::$filters[ Optml_Settings::FILTER_TYPE_LAZYLOAD ][ Optml_Settings::FILTER_FILENAME ] ) ) {
 			return false;
 		}
+		$url = strtok( $url, '?' );
+
```

Comment:
can you send a pr just with this in a separate PR, along with the test? rest of them seems not ready yet.

Reply:
Solved in #147